README.md

```diff
@@ -1 +1,74 @@
-# Capstone-2
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+#  Capstone 2
+
+> The JavaScript capstone project is about building your own web application based on an external API. We will select an API that provides data about a topic that we like and then build the webapp around it. The webapp will have 2 user interfaces.
+
+## Live preview
+
+Take a look at the Movies page we Create => [link](https://juanse7793.github.io/Capstone-2/)
+
+## Built With
+
+- HTML & CSS
+- JavaScript
+- Webpack
+- GitFlow
+- API
+
+## Run it
+
+Use the following steps to run this Project locally:
+
+- Create a directory
+
+- Open the terminal
+
+- Run this command:
+`git clone git@github.com:Juanse7793/Capstone-2.git`
+
+- Enter in to the folder:
+`cd Capstone-2`
+
+- Run `npm i` to install all dependencies
+- Run `npm run` build to bundle the project with webpack, and
+- Run `npm start` to launch the application
+
+- Open the folder with a code editor (VS Code preferred)
+
```

Comment:
- [ ] I would recommend adding the command to test the application as well. Hint: `npm test`.

Reply:
thank you, it's done :+1: